.styleci.yml

```diff
@@ -1,4 +1,6 @@
-preset: psr2
+preset: recommended
 
-enabled:
-  - return
+disabled:
+  - align_double_arrow
+  - concat_without_spaces
+  - no_multiline_whitespace_before_semicolons
```

Comment:
There is a less aggressive version of this fixer you might want to replace it with. I can't remember it's name, but I think it does something along the lines of removing the whitespace, but not in multiline contexts.

Reply:
Do you mean this one "no_singleline_whitespace_before_semicolons" ?

Reply:
Yeh, that sounds right. :)

Reply:
So I keep this one disabled and enable the singleline one, right?